### PR TITLE
pythonPackages.simplekml: Init at 1.3.1

### PR DESCRIPTION
**Pull request by rvolosatovs**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/simplekml/default.nix [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{ lib , buildPythonPackage , fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "simplekml";
+  version = "1.3.1";
+  name = "simplekml-${version}";
```
Comment (FRidh) [inline pkgs/development/python-modules/simplekml/default.nix]: The `name` attribute is added by `buildPython*` and should therefore be removed.

**Comment by FRidh** [comment]
@GrahamcOfBorg build pythonPackages.simplekml python3Packages.simplekml

**Review by FRidh — CHANGES_REQUESTED** [review]

**Comment by rvolosatovs** [comment]
@GrahamcOfBorg build pythonPackages.simplekml python3Packages.simplekml

**Comment by FRidh** [comment]
@GrahamcOfBorg build pythonPackages.simplekml python3Packages.simplekml

**Comment by rvolosatovs** [comment]
@FRidh

**Comment by rvolosatovs** [comment]
@FRidh, done

**Comment by worldofpeace** [comment]
@GrahamcOfBorg build pythonPackages.simplekml python3Packages.simplekml

**Pull request merged**